app/assets/stylesheets/administrate/library/_mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin administrate-clearfix {
```

Comment:
Can we name this file `_clearfix.scss` to match the name of the mixin it houses? It makes finding things super duper easy.